examples/jsm/postprocessing/OutlinePass.js

```diff
@@ -46,8 +46,6 @@ class OutlinePass extends Pass {
 		const resx = Math.round( this.resolution.x / this.downSampleRatio );
 		const resy = Math.round( this.resolution.y / this.downSampleRatio );
 
-		this.maskBufferMaterial = new MeshBasicMaterial( { color: 0xffffff } );
-		this.maskBufferMaterial.side = DoubleSide;
```

Comment:
How come you removed these? 🤔

Reply:
`this.maskBufferMaterial` is unused in this example which makes me confuse for a while. So in my opinion, I think it's more reasonable to remove it.

Reply:
Any idea of why that code was there to begin with?
You can look at the commit history to investigate.

Reply:
I've checked the history and found it begin with the initial commit and `this.maskBufferMaterial` is also unused in the initial version.

Here's the link: https://github.com/mrdoob/three.js/commit/9e6f4306b86fe31c91b4fa762cf5b61b864600bf#diff-9b3033a5e289a75f2da1383ac047c096c8737a31f3d70ccdeebd2b6d2faf3dcd